runelite-client/src/main/java/net/runelite/client/plugins/combatlevel/CombatLevelPlugin.java

```diff
@@ -25,33 +25,72 @@
 package net.runelite.client.plugins.combatlevel;
 
 import com.google.common.eventbus.Subscribe;
+import com.google.inject.Provides;
 import java.text.DecimalFormat;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
 import javax.inject.Inject;
 import net.runelite.api.Client;
 import net.runelite.api.Experience;
 import net.runelite.api.GameState;
 import net.runelite.api.Skill;
+import net.runelite.api.WorldType;
+import net.runelite.api.events.ConfigChanged;
 import net.runelite.api.events.GameTick;
+import net.runelite.api.events.WidgetTextChanged;
 import net.runelite.api.widgets.Widget;
 import net.runelite.api.widgets.WidgetInfo;
+import net.runelite.client.config.ConfigManager;
 import net.runelite.client.plugins.Plugin;
 import net.runelite.client.plugins.PluginDescriptor;
 
 @PluginDescriptor(
 	name = "Combat Level",
-	description = "Show a more accurate combat level in Combat Options panel"
+	description = "Show a more accurate combat level in Combat Options panel and other combat level functions",
+	tags = {"wilderness", "attack", "range"}
 )
 public class CombatLevelPlugin extends Plugin
 {
-	private final DecimalFormat decimalFormat = new DecimalFormat("#.###");
+	private static final DecimalFormat decimalFormat = new DecimalFormat("#.###");
```

Comment:
UPPER_CASE

Reply:
Done, and changed my inspections settings so you shouldn't have to bother me about this again. :laughing: